src/DataMiner/SLNetTypes/Responses/GetProtocolInfoResponseMessage.cs

```diff
@@ -1,12 +1,1127 @@
+		public SearchControl[] SearchControls { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating whether this protocol contains at least one active param that will be indexed as DynamicData.
+		/// </summary>
+		/// <value><c>true</c> if this protocol contains at least one active param that will be indexed as DynamicData.</value>
+		public bool IsIndexing { get; set; }
+
+		/// <summary>
+		/// The protocol description.
+		/// </summary>
+		public String Description;
+
+		/// <summary>
+		/// The protocol name.
+		/// </summary>
+		public String Name;
+
+		/// <summary>
+		/// Gets or sets the provider.
+		/// </summary>
+		/// <value>The provider</value>
+		public String Provider { get; set; }
+
+		/// <summary>
+		/// Gets or sets the advanced security.
+		/// </summary>
+		/// <value>The advanced security.</value>
+		public String AdvancedSecurity { get; set; }
+
+		/// <summary>
+		/// Gets or sets the name of the parent protocol for which the protocol was exported. (For exported protocols).
+		/// </summary>
+		/// <value>For exported protocols, the name of the parent protocol for which the protocol was exported.</value>
+		public string ParentProtocolName { get; set; }
+
+		/// <summary>
+		/// The protocol version.
+		/// </summary>
+		public String Version;
+
+		/// <summary>
+		/// Gets or sets the protocol version history.
+		/// </summary>
+		/// <value>The protocol version history.</value>
+		public VersionHistory VersionHistory { get; set; }
+
+		/// <summary>
+		/// Gets or sets the process automation info.
+		/// </summary>
+		/// <value>The process automation info.</value>
+		public ProcessAutomationInfo ProcessAutomationInfo { get; set; }
+
+		/// <summary>
+		/// The vendor.
+		/// </summary>
+		public String Vendor;
+
+		/// <summary>
+		/// The list of available protocol functions.
+		/// </summary>
+		public List<FunctionDefinition> ProtocolFunctions { get; set; }
+
+		/// <summary>
+		/// The protocol type.
+		/// </summary>
+		public String Type;
+
+		/// <summary>
+		/// Gets or sets the protocol type.
+		/// </summary>
+		/// <value>The protocol type.</value>
+		public ProtocolType ProtocolType
+		{
+			get; set;
+		}
+
+		/// <summary>
+		/// Gets the protocol types.
+		/// </summary>
+		/// <value>The protocol types.</value>
+		public ProtocolType[] ProtocolTypes
+		{
+			get;
+		}
+
+		/// <summary>
+		/// Checks the main type and advanced types if it matches the specified value.
+		/// </summary>
+		/// <param name="typeToCheck">The type to check (e.g. serial).</param>
+		/// <returns><c>true</c> if the main type or one of the advanced types matches the specified value.</returns>
+		public bool IsProtocolType(ProtocolType typeToCheck)
+		{
+			return ProtocolTypes.Contains(typeToCheck);
+		}
+
+		/// <summary>
+		/// The main connection name.
+		/// </summary>
+		public String MainConnectionName;
+
+		/// <summary>
+		/// Converts the specified type to <see cref="¨ProtocolType"/>.
```

Comment:
Is this "¨" a mistake?
```suggestion
		/// Converts the specified type to <see cref="ProtocolType"/>.
```